crates/kdc/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "kdc"
+version = "0.1.0"
+edition = "2024"
+
```

Comment:
note: If you don’t want to publish the initial version right now, you should add publish = false here (and remove that later when publishing is intended)